### PR TITLE
#404 Organisms - FOPaperRow

### PR DESCRIPTION
**Pull request by bryantobing12**
### Detailed description of changes

- Adding `FOPaperRow` Component
- Adding `FOPaperRow` Stories
- Adding `FOPaperRow` Test

### Recommendations on how to test the changes

- Check if the component match the design

### Checklist for developer

- [x] The code builds clean without any errors or warnings
- [x] The code does not contain commented out code
- [x] The code does not log anything to console
- [x] I have added unit test(s) to cover new code and successfully executed ran it
- [x] I have thoroughly tested the new code and any adjacent features it may affect

### Link to related issue(s)

https://app.zenhub.com/workspaces/flick-614155581501e1000f356a5d/issues/hzn-one/tms/404

#### Before

N / A

#### After
Figma
image

Result
image

### CODE REVIEW

File: src/Components/Organisms/FOPaperRow/index.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+  useEffect(() => {
+    if (actionRef.current?.offsetWidth) {
+      setActionWidth(`${actionRef.current.offsetWidth}px`);
+    }
+  }, [actionRef.current?.offsetWidth]);
+
+  return (
+    <Paper className="flex-paper-2" sx={{ position: "relative" }}>
+      <Box
```
Comment (gilangdarmawan) [inline src/Components/Organisms/FOPaperRow/index.tsx:23]: better pakai grid

File: src/Components/Organisms/FOPaperRow/index.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        display="flex"
+        gap={3}
+        sx={{ width: `calc(100% - ${actionWidth})` }}
+        overflow="auto"
+      >
+        {items.map((item, idx) => {
+          return (
+            <Box key={idx}>
```
Comment (gilangdarmawan) [inline src/Components/Organisms/FOPaperRow/index.tsx:31]: set grid item auto

File: src/Components/Organisms/FOPaperRow/index.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                </FATypography>
+              ) : (
+                <Box>{item.children}</Box>
+              )}
+            </Box>
+          );
+        })}
+      </Box>
+      {actions && (
+        <Box
```
Comment (gilangdarmawan) [inline src/Components/Organisms/FOPaperRow/index.tsx:56]: better use grid system. [here's](https://mui.com/components/grid/#variable-width-content) the reference

**Review by gilangdarmawan — CHANGES_REQUESTED** [review]

**Comment by ardiwine** [comment]
jarak antara column nya kayaknya belum sesuai ya ?

**Comment by bryantobing12** [comment]
Updated @gilangdarmawan  @ardiwine

**Review by gilangdarmawan — APPROVED** [review]

**Pull request merged**